packages/rolldown/src/options/output-options.ts

```diff
@@ -70,9 +70,10 @@ export interface OutputOptions {
    * Control comments in the output.
    *
    * - `none`: no comments
+   * - `preserve`: preserve all comments except legal comments
```

Comment:
I'm not sure if this description is accurate, as I wrote it based on the execution logic.

https://github.com/rolldown/rolldown/blob/f1c8656cd231ddff20ded658665a987a78e41fd6/crates/rolldown_ecmascript/src/ecma_compiler.rs#L57-L60